onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -109,24 +109,46 @@ def quantize_nparray(qType, arr, scale, zero_point, low=None, high=None):
     return arr_fp32.astype(dtype)
 
 
-def compute_scale_zp(rmin, rmax, qType, quantize_range):
-    if qType == onnx_proto.TensorProto.INT8:
-        max_range = max(abs(rmin), abs(rmax))
-        scale = (float(max_range) * 2) / quantize_range if max_range > 0 else 1
-        zero_point = 0
-    elif qType == onnx_proto.TensorProto.UINT8:
-        scale = (float(rmax) - rmin) / quantize_range if rmin != rmax else 1
-        zero_point = round((0 - rmin) / scale)  # round to nearest integer
-    else:
-        raise ValueError("Unexpected data type {} requested. Only INT8 and UINT8 are supported.".format(qType))
+def compute_scale_zp(rmin, rmax, qmin, qmax, symmetrize=False):
```

Comment:
There is an update to support symmetric. https://github.com/microsoft/onnxruntime/blob/e9057d2e498e0c9233669cae11223bf4f6896ba6/onnxruntime/python/tools/quantization/quant_utils.py#L112